Document/0x06d-Testing-Data-Storage.md

```diff
@@ -415,7 +415,26 @@ Time: 0.013s
 
 ##### Searching for Cache Databases
 
-iOS applications can store data in cache databases. These databases contain data such as web requests and responses. This data can be sensitive, if tokens, usernames or other information is cached. To display the cache go to the app path and go to `/Library/Caches/{Bundle Identifier}`. The WebKit cache is being stored in the Cache.db file. Objection can interact with it, as it is a normal SQLite database.
+By Default NSURLSession stores data in Cache.db database. This database contain data such as API requests and responses. This database can contain sensitive data, if tokens, usernames or any other sensitive information has been cached. To find the cache files go to the app path and go to `/Library/Caches/{Bundle Identifier}`. The WebKit cache is also being stored in the Cache.db file. Objection can interact with it, as it is a normal SQLite database.
+
+###### Recommendations
+
+It is recommended to disable Caching this data as it may contain sensitive data in the URL parameters or request response. It may cause the violation of the [MSTG-Storage-1](https://github.com/OWASP/owasp-masvs/blob/master/Document/0x07-V2-Data_Storage_and_Privacy_requirements.md "OWASP MASVS").
+
+1. It is recommended to remove Cached responses after logout. It can be done with the provided function by Apple [removeAllCachedResponses](https://developer.apple.com/documentation/foundation/urlcache/1417802-removeallcachedresponses "URLCache removeAllCachedResponses")
```

Comment:
```suggestion
1. It is recommended to remove Cached responses after logout. This can be done with the provided method by Apple called [`removeAllCachedResponses`](https://developer.apple.com/documentation/foundation/urlcache/1417802-removeallcachedresponses "URLCache removeAllCachedResponses")
```